packages/tsx/package.json

```diff
@@ -27,7 +27,7 @@
     "@nodejs-loaders/parse-filename": "1.0.0"
   },
   "peerDependencies": {
-    "esbuild": "^0.25.0"
+    "esbuild": ">=0.19.5 <=0.25.0"
```

Comment:
The range is correct, but it should still be a separate PR 😅

Reply:
what ? I had remove ts ignore thing and change pr title

Reply:
OH! sorry, GH was still showing them here 😵